### ISSUE TITLE
Now must use secure connections only?

### ISSUE THREAD

**Issue by Jibun-no-Kage** [hobbyquaker/lgtv2#49]
In January 2023, LG disabled http access?  Now must use secure https connections only?  So wss: and port 3001 only?

**Comment by hobbyquaker** [comment]
Yes - since the 2023 firmware wave TVs only accept `wss://<tv>:3001`. This is fixed on `master` (tagged `v1.8.0`, see the [CHANGELOG](https://github.com/hobbyquaker/lgtv2/blob/master/CHANGELOG.md)); the npm release follows shortly.

With `new LGTV({host: '192.168.1.20'})` the module now tries `wss://host:3001` first and falls back to `ws://host:3000` for old TVs automatically; a full `url` still works. Certificate verification against public CAs is off by default (LG uses its own private CA); `verifyCert: 'lg'` pins LG's TV certificate if you want that.
